# Russian translation for linuxmint
# Copyright (c) 2013 Rosetta Contributors and Canonical Ltd 2013
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2013.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2023-11-27 14:33+0000\n"
"PO-Revision-Date: 2022-09-28 11:30+0000\n"
"Last-Translator: Dmitriy Kulikov <Unknown>\n"
"Language-Team: Russian <ru@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2025-01-06 11:56+0000\n"
"X-Generator: Launchpad (build 6394e03793719e8d73f5a60b5439440e693c92f1)\n"

#: lib/mintstick.py:312
msgid "The USB stick was formatted successfully."
msgstr "USB-накопитель успешно отформатирован."

#: lib/mintstick.py:315
#, python-format
msgid "An error occured while creating a partition on %s."
msgstr "При создании раздела на %s возникла ошибка."

#: lib/mintstick.py:317 lib/mintstick.py:427
msgid "Authentication Error."
msgstr "Ошибка аутентификации."

#: lib/mintstick.py:322 lib/mintstick.py:433
msgid "An error occurred."
msgstr "Произошла ошибка."

#: lib/mintstick.py:420
msgid "The image was successfully written."
msgstr "Образ был успешно записан."

#: lib/mintstick.py:423
msgid "Not enough space on the USB stick."
msgstr "Недостаточно свободного места на USB-накопителе"

#: lib/mintstick.py:425
msgid "An error occured while copying the image."
msgstr "В процессе копирования образа произошла ошибка."

#: lib/verify.py:54
msgid "bytes"
msgstr "байт"

#: lib/verify.py:54
msgid "KB"
msgstr "КБ"

#: lib/verify.py:54
msgid "MB"
msgstr "МБ"

#: lib/verify.py:54
msgid "GB"
msgstr "ГБ"

#: lib/verify.py:54
msgid "TB"
msgstr "ТБ"

#: lib/verify.py:87
msgid "ISO images"
msgstr "ISO-образы"

#: lib/verify.py:91
msgid "Checksum files"
msgstr "Файл контрольных сумм"

#: lib/verify.py:95
msgid "GPG signatures"
msgstr "GPG сигнатуры"

#: lib/verify.py:107 lib/verify.py:108 lib/verify.py:109
msgid "Calculating..."
msgstr "Вычисление..."

#: lib/verify.py:167
msgid "No volume ID found"
msgstr "Идентификатор тома не найден"

#: lib/verify.py:208
msgid "The checksum is correct"
msgstr "Контрольная сумма верна"

#: lib/verify.py:209
msgid ""
"The checksum is correct but the authenticity of the sum was not verified."
msgstr "Контрольная сумма верна, но ее подлинность не проверялась."

#: lib/verify.py:211
msgid "Checksum mismatch"
msgstr "Несовпадение контрольной суммы"

#: lib/verify.py:212
msgid "Download the ISO image again. Its checksum does not match."
msgstr "Загрузите ISO-образ повторно. Его контрольная сумма не корректна"

#: lib/verify.py:223
msgid "The sums file could not be downloaded. Check the URL."
msgstr "Файл с суммами не может быть загружен. Проверьте правильность URL."

#: lib/verify.py:232
msgid "The gpg file could not be downloaded. Check the URL."
msgstr "Файл gpg не может быть загружен. Проверьте правильность URL."

#: lib/verify.py:243
msgid "The sums file could not be checked."
msgstr "Файл с суммами не может быть проверен."

#: lib/verify.py:250
msgid "The gpg file could not be checked."
msgstr "Файл gpg не может быть проверен."

#: lib/verify.py:260
msgid "Integrity check failed"
msgstr "Ошибка проверки целостности"

#: lib/verify.py:269
msgid "The SHA256 sums file is not signed."
msgstr "Файл суммы SHA256 не подписан."

#: lib/verify.py:273
#, python-format
msgid "Signed by: %s"
msgstr "Подписано: %s"

#: lib/verify.py:287
msgid "Unknown signature"
msgstr "Неизвестная подпись"

#: lib/verify.py:288
msgid "Key not found on keyserver."
msgstr "Ключ не найден на сервере."

#: lib/verify.py:299 lib/verify.py:305
msgid "Everything looks good!"
msgstr "Все выглядит хорошо!"

#: lib/verify.py:300
msgid "This is an official ISO image."
msgstr "Это официальный ISO-образ."

#: lib/verify.py:306
msgid "This ISO image is verified by a trusted signature."
msgstr "Данный ISO-образ подтвержден доверенной подписью."

#: lib/verify.py:309
msgid "If you trust the signature you can trust the ISO."
msgstr "Если вы доверяете подписи, вы можете доверять ISO-образу."

#: lib/verify.py:310
msgid "Untrusted signature"
msgstr "Недоверенная подпись"

#: lib/verify.py:311
msgid "This ISO image is verified by an untrusted signature."
msgstr "Данный ISO-образ подтвержден недоверенной подписью."

#: lib/verify.py:316
msgid "An error occurred"
msgstr "Произошла ошибка"

#: lib/verify.py:327
msgid "The SHA256 sum of the ISO image is incorrect."
msgstr "Сумма SHA256 ISO-образа неверна."

#: lib/verify.py:328
msgid "The SHA256 sums file does not contain sums for this ISO image."
msgstr "Файл сумм SHA256 не содержит контрольных сумм для этого ISO-образа."

#: generate_additional_files.py:22 generate_additional_files.py:34
#: share/mintstick/mintstick.ui.h:38
msgid "USB Image Writer"
msgstr "Запись образа на USB-накопитель"

#: generate_additional_files.py:22 generate_additional_files.py:34
#: generate_additional_files.py:67 share/mintstick/mintstick.ui.h:39
msgid "Make a bootable USB stick"
msgstr "Создание загрузочного USB-носителя"

#: generate_additional_files.py:46 generate_additional_files.py:58
#: share/mintstick/mintstick.ui.h:27
msgid "USB Stick Formatter"
msgstr "Форматирование USB-флеш-накопителя"

#: generate_additional_files.py:46 generate_additional_files.py:58
#: generate_additional_files.py:77 share/mintstick/mintstick.ui.h:28
msgid "Format a USB stick"
msgstr "Форматировать USB-накопитель"

#: generate_additional_files.py:67
msgid "Make bootable USB stick"
msgstr "Создание загрузочного USB-накопителя"

#: generate_additional_files.py:77 share/mintstick/mintstick.ui.h:26
msgid "Format"
msgstr "Форматировать"

#: generate_additional_files.py:86 share/mintstick/mintstick.ui.h:32
#: share/mintstick/verify.ui.h:7
msgid "Verify"
msgstr "Проверить"

#: generate_additional_files.py:86 share/mintstick/mintstick.ui.h:33
msgid "Verify the authenticity and integrity of the image"
msgstr "Проверка подлинности и целостности образа"

#: generate_additional_files.py:115 generate_additional_files.py:144
msgid ""
"This will destroy all data on the USB stick, are you sure you want to "
"proceed?"
msgstr ""
"Вся информмация  на USB-флеш-накопителе будет уничтожена. Продолжить?"

#: share/mintstick/mintstick.ui.h:1
msgid "USB stick:"
msgstr "USB-накопитель:"

#: share/mintstick/mintstick.ui.h:2
msgid "Filesystem:"
msgstr "Файловая система:"

#: share/mintstick/mintstick.ui.h:3
msgid "Select a USB stick"
msgstr "Выберите USB-флеш-накопитель"

#: share/mintstick/mintstick.ui.h:4
msgid ""
"FAT32 \n"
"  + Compatible everywhere.\n"
"  -  Cannot handle files larger than 4GB.\n"
"\n"
"exFAT\n"
"  + Compatible almost everywhere.\n"
"  + Can handle files larger than 4GB.\n"
"  -  Not as compatible as FAT32.\n"
"\n"
"NTFS \n"
"  + Compatible with Windows.\n"
"  -  Not compatible with Mac and most hardware devices.\n"
"  -  Occasional compatibility problems with Linux (NTFS is proprietary and "
"reverse engineered).\n"
"\n"
"EXT4 \n"
"  + Modern, stable, fast, journalized.\n"
"  + Supports Linux file permissions.\n"
"  -  Not compatible with Windows, Mac and most hardware devices.\n"
msgstr ""
"FAT32 \n"
"  + Совместима со всеми ОС и аппаратными устройствами.\n"
"  -  Нельзя хранить файлы более 4Гб.\n"
"\n"
"exFAT\n"
"  + Совместима почти со всеми ОС и аппаратными устройствами.\n"
"  + Возможно хранить файлы более 4Гб.\n"
"  -  Не такая широко поддерживаемая, как FAT32.\n"
"\n"
"NTFS \n"
"  + Совместима с Windows.\n"
"  -  Не совместима с Mac и большинством аппаратных устройств.\n"
"  -  Незначительные проблемы совместимости с Linux (NTFS является "
"проприетарной файловой системой).\n"
"\n"
"EXT4 \n"
"  + Современная, стабильная, быстрая, журнализируемая.\n"
"  + Поддерживает файловые разрешения Linux.\n"
"  -  Не совместима с Windows, Mac и большинством аппаратных устройств.\n"

#: share/mintstick/mintstick.ui.h:23
msgid "Volume label:"
msgstr "Метка тома:"

#: share/mintstick/mintstick.ui.h:24
msgid "Choose a name for your USB Stick"
msgstr "Выберите метку для USB-накопителя"

#: share/mintstick/mintstick.ui.h:25
msgid "USB Stick"
msgstr "USB-флеш-накопитель"

#: share/mintstick/mintstick.ui.h:29
msgid "ISO image:"
msgstr "ISO-образ:"

#: share/mintstick/mintstick.ui.h:30
msgid "Select an image"
msgstr "Выберите образ"

#: share/mintstick/mintstick.ui.h:31
msgid "Select Image"
msgstr "Выбрать образ"

#: share/mintstick/mintstick.ui.h:34
msgid "Write"
msgstr "Записать"

#: share/mintstick/mintstick.ui.h:35
msgid "This ISO looks like a Windows image."
msgstr "Даннй ISO-образ похож на образ Windows."

#: share/mintstick/mintstick.ui.h:36
msgid "Windows images require special processing."
msgstr "Образы Windows требуют специальной обработки."

#: share/mintstick/mintstick.ui.h:37
msgid "More information"
msgstr "Подробнее"

#: share/mintstick/verify.ui.h:1
msgid "ISO:"
msgstr "ISO:"

#: share/mintstick/verify.ui.h:2
msgid "Volume:"
msgstr "Том:"

#: share/mintstick/verify.ui.h:3
msgid "SHA256sum:"
msgstr "SHA256sum:"

#: share/mintstick/verify.ui.h:4
msgid "Size:"
msgstr "Размер:"

#: share/mintstick/verify.ui.h:5
msgid "SHA256 sums file:"
msgstr "Файл сумм SHA256:"

#: share/mintstick/verify.ui.h:6
msgid "GPG signed file:"
msgstr "Файл, подписанный GPG:"

#: share/mintstick/verify.ui.h:8
msgid "URLs"
msgstr "Адреса URL"

#: share/mintstick/verify.ui.h:9
msgid "SHA256 sums file"
msgstr "Файл сумм SHA256"

#: share/mintstick/verify.ui.h:10
msgid "GPG signed file"
msgstr "Файл, подписанный GPG"

#: share/mintstick/verify.ui.h:11
msgid "Local files"
msgstr "Локальные файлы"

#: share/mintstick/verify.ui.h:12
msgid "SHA256 sum"
msgstr "Сумма SHA256"

#: share/mintstick/verify.ui.h:13
msgid "Check"
msgstr "Проверить"

#: share/mintstick/verify.ui.h:14
msgid "Checksum"
msgstr "Контрольная сумма"

#: share/mintstick/verify.ui.h:15
msgid "Go back"
msgstr "Назад"

#: share/mintstick/verify.ui.h:16
msgid "ISO Verification"
msgstr "Верификация ISO-образа"
